bit-buffer.js

```diff
@@ -59,16 +59,18 @@ BitView.prototype.getBits = function (offset, bits, signed) {
 
 	var value = 0;
 	for (var i = 0; i < bits;) {
-		var read;
-
-		// Read an entire byte if we can.
-		if ((bits - i) >= 8 && ((offset & 7) === 0)) {
-			value |= (this._view[offset >> 3] << i);
-			read = 8;
-		} else {
-			value |= (this._getBit(offset) << i);
-			read = 1;
-		}
+		var toRead = bits - i;
```

Comment:
`var remaining` would avoid the confusion with the subsequent read variable name.